### PR TITLE
Added centralite micro-door sensor 3323-D

### PR DESCRIPTION
**Pull request by adik6555**

### CODE REVIEW

**Comment by adik6555** [comment]
Added Centralite 3323-D

File: devices.js [hunk 1]
```diff
@@ -1961,6 +1961,16 @@ const devices = [
...
         },
     },
+    // Centralite Micro Door sensor
+    {
+        zigbeeModel: ['3323-D'],
+        model: '3323-D',
+        vendor: 'Centralite',
+        description: 'Micro Door sensor with temp and humidity',
+        supports: 'open/close,temperature and humidity',
+        fromZigbee: [],
```
Comment (Koenkk) [inline devices.js]: I expect at least one converter here, did you get this device working?

**Comment by adik6555** [comment]
I thought i had to put it here first and then work on converters, just as it is in the tutorial.

**Comment by Koenkk** [comment]
Yes, that's indeed the first step, however before this can get merged it should also work (which it can't without converters).

Please add the converters, in case you require help please open an issue.

**Comment by Koenkk** [comment]
When triggering the sensor (open/close door). Do you see any logging like `No converter available for...`?

**Pull request closed without merging**